src/chromatix/functional/sources.py

```diff
@@ -114,13 +114,15 @@ def plane_wave(
         phase: The phase of the plane wave in radians, defaults to 0.0.
         pupil: If provided, will be called on the field to apply a pupil.
         k: If provided, defines the orientation of the plane wave. Should be an
-            array of shape `[2 H W]`. If provided, ``phase`` is ignored.
+            array of shape `[2,]`. If provided, ``phase`` is ignored.
```

Comment:
Can we actually ask for a vector in [y x] here and not reverse it later? And specify that order in this docstring.